submissions/sofiichuk/oop-exercise/index.js

```diff
@@ -0,0 +1,65 @@
+class Animal {
+    constructor(name, gender, saying) {
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+    }
+}
+
+class Human extends Animal {
+    constructor(name, gender, saying, legs = 2, arms = 2) {
+        super(name, gender, saying);
+        this.legs = legs;
+        this.arms = arms;
+    }
+}
+
+class Pet extends Animal {
+    constructor(name, gender, saying, legs = 4) {
+        super(name, gender, saying);
+        this.legs = legs;
```

Comment:
What is so special about `legs` in comparison with e.g. `name` so that it is not generalized into the base class?

Reply:
moved them to the base class